OrbitGl/ThreadTrack.cpp

```diff
@@ -168,7 +172,9 @@ void ThreadTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, Picking
 
 void ThreadTrack::SetTrackColor(Color color) {
   ScopeLock lock(mutex_);
-  event_track_->SetColor(color);
+  if (thread_id_ != SamplingProfiler::kAllTracepointsFakeTid) {
```

Comment:
Same as above, it would be better to avoid introducing a dependency to SamplingProfiler.